src/bidiMapper/domains/context/browsingContextImpl.ts

```diff
@@ -190,6 +190,17 @@ export class BrowsingContextImpl {
     return this.#parentId === null;
   }
 
+  get top(): BrowsingContextImpl {
```

Comment:
nit: Either rename to `get topLevelContext()` or `getTopLevelContext()`, or add JSDoc (or maybe both).